src/CMakeLists.txt

```diff
@@ -44,23 +44,27 @@ function(hdr_histogram_add_library NAME LIBRARY_TYPE)
         PUBLIC
             $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}>
             $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}>)
-    install(
-        TARGETS ${NAME}
-        EXPORT ${PROJECT_NAME}-targets
-        DESTINATION ${CMAKE_INSTALL_LIBDIR})
+    if(DO_INSTALL)
+        install(
+            TARGETS ${NAME}
+            EXPORT ${PROJECT_NAME}-targets
+            DESTINATION ${CMAKE_INSTALL_LIBDIR})
+    endif()
 endfunction()
 
 option(HDR_HISTOGRAM_BUILD_SHARED "Build shared library" ON)
+option(HDR_HISTOGRAM_BUILD_SHARED "Install shared library" ON)
```

Comment:
@remicollet, Should this have been `HDR_HISTOGRAM_INSTALL_SHARED`? Same two lines below. Otherwise it's a duplicating option name right now.